src/pages/organize/[orgId]/people/[personId]/index.tsx

```diff
@@ -112,6 +110,15 @@ const PersonProfilePage: PageWithLayout<PersonPageProps> = (props) => {
             )}
           </ZetkinQuery>
         </Grid>
+        <Grid item lg={4} xs={12}>
+          <PersonJourneysCard
+            orgId={orgId as string}
+            personId={personId as string}
+          />
```

Comment:
Similar to an above comment, you could just pass in the entire person and get the org id out of there right?

Reply:
The person object does not contain an organization, because a person can be in multiple organizations. So it would need to be retrieved from the router or something.

But yeah, this is annoying and I'd like to come up with a better way to handle organization IDs in general.